Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2755 @@
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="IsmBuiltUpSectionComponent_BuiltUpSection" strength="referencing" strengthDirection="forward" modifier="Sealed">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="is part of" polymorphic="false">
+            <Class class="IsmBuiltUpSectionComponent" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="has components" polymorphic="false">
+            <Class class="IsmBuiltUpSection" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Components</SourceDisplayLabel>
+                <TargetDisplayLabel>Section</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="IsmBuiltUpSectionComponent_Section" strength="referencing" strengthDirection="forward" modifier="Sealed">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses section" polymorphic="false">
+            <Class class="IsmBuiltUpSectionComponent" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is part of built up section" polymorphic="true">
+            <Class class="IsmConstantSection" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Components</SourceDisplayLabel>
+                <TargetDisplayLabel>Section</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IsmTableSection" displayLabel="Table Section" modifier="Sealed">
+        <BaseClass>IsmFixedParametricSection</BaseClass>
+        <ECProperty propertyName="StandardsOrganization" typeName="string" displayLabel="Standards' Organization" description="The standards organization that defines classification for this Section." />
+        <ECProperty propertyName="SectionName" typeName="string" displayLabel="Section Name" description="Section name." />
+        <ECProperty propertyName="Manufacturer" typeName="string" description="Manufacturer." />
+        <ECProperty propertyName="Revision" typeName="string" displayLabel="Revision" description="Revision." />
+    </ECEntityClass>
+
+    <ECEntityClass typeName="IsmParametricSection" displayLabel="Parametric Section" modifier="None">
+        <BaseClass>IsmFixedParametricSection</BaseClass>
+    </ECEntityClass>
+
+    <ECEnumeration typeName="SteelJoistBearing" backingTypeName="int" isStrict="true" description="Steel Joist Bearing.">
+        <ECEnumerator value="10" name="Bottom" displayLabel="Bottom" description="Bottom." />
+        <ECEnumerator value="20" name="Top" displayLabel="Top" description="Top." />
```

Comment:
unnecessary display label and description

Reply:
Fixed